pkg/cdclog/decoder_test.go

```diff
@@ -15,9 +15,11 @@ package cdclog
 
 import (
 	"encoding/binary"
+	"encoding/json"
 	"testing"
 
-	"github.com/pingcap/check"
+	"github.com/pingcap/parser/mysql"
+	"gopkg.in/check.v1"
```

Comment:
are you importing the wrong `check` package

Reply:
I'll fix it